src/datashare/azext_datashare/generated/_validators.py

```diff
@@ -0,0 +1,18 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def example_name_or_id_validator(cmd, namespace):

```

Comment:
Unused code.

Reply:
saw this in another PR, pls feedback to codegen team to remove it